src/Eval.hs

```diff
@@ -204,6 +204,7 @@ eval ctx xobj@(XObj o info ty) preference resolver =
             (WithPat (SymPat sym path) forms) -> specialCommandWith ctx sym path forms
             (DoPat forms) -> evaluateSideEffects forms
             (WhilePat cond body) -> specialCommandWhile ctx cond body
+            (SetPat iden value) -> specialCommandSet ctx (iden:[value])
```

Comment:
Is this reordering an unrelated cleanup or important for this change?

Reply:
Yeah, this was just a small change for readability, I can undo it if we want a cleaner history.

Reply:
No, that’s fine with me, I just wanted to check whether I was missing something!